<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="small_gear">
    <static>false</static>
    <allow_auto_disable>false</allow_auto_disable>

    <link name="small_gear">
      <inertial auto="true"/>
      
      <visual name="small_gear_visual">
        <geometry>
          <mesh>
            <uri>model://demo_parts/small_gear/meshes/small_gear.stl</uri>
          </mesh>
        </geometry>
        <material>
          <ambient>1.0 1.0 1.0 1.0</ambient>
          <diffuse>1.0 1.0 1.0 1.0</diffuse>
        </material>
      </visual>

      <collision name="small_gear_collision">
        <density>10000</density>
        <pose>0 0 0 0 0 0</pose>
        <geometry>
          <mesh>
            <uri>model://demo_parts/small_gear/meshes/small_gear.stl</uri>
          </mesh>
        </geometry>
        <max_contacts>5</max_contacts>
      </collision>
    </link>
  </model>
</sdf>